cluster/manifests/wiz/sensor-daemonset.yaml

```diff
@@ -0,0 +1,203 @@
+{{ if eq .Cluster.ConfigItems.wiz_enable_runtime_sensor "true"}}
+---
+# Source: wiz-sensor/templates/daemonset.yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: wiz-sensor
+  labels:
+    helm.sh/chart: wiz-sensor-1.0.4760
+    application: "wiz"
+    component: "sensor"
+  namespace: wiz
+spec:
+  selector:
+    matchLabels:
+      application: "wiz"
+      component: "sensor"
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        helm.sh/chart: wiz-sensor-1.0.4760
+        application: "wiz"
+        component: "sensor"
+      annotations:
+        container.apparmor.security.beta.kubernetes.io/wiz-sensor: unconfined
+        cluster-autoscaler.kubernetes.io/enable-ds-eviction: "true"
+        node-ready.cluster.zalando.org/exclude: "true"
+    spec:
+      serviceAccountName: wiz-sensor
```

Comment:
This service account should be added to the privileged list here: https://github.com/zalando-incubator/kubernetes-on-aws/blob/5f4fdd525d1fbabd0fbd717e387367c8843528ca/cluster/manifests/01-admission-control/config.yaml#L95-L113

it should ofc. only be added if `eq .Cluster.ConfigItems.wiz_enable_runtime_sensor "true"`